{"uuid": "52b97c3a-3cbf-466c-b4ef-fbc191c28e0e", "children": ["20ac5a56-31dd-47a1-86ed-13b62c8fd419"], "befores": [{"name": "with_login_cpc", "status": "passed", "start": 1568472753244, "stop": 1568472759786}], "afters": [{"name": "with_login_cpc::0", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.InvalidSessionIdException: Message: invalid session id\n\n", "trace": "  File \"c:\\users\\dajun\\documents\\caipiao\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 185, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"c:\\users\\dajun\\documents\\caipiao\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 786, in _teardown_yield_fixture\n    next(it)\n  File \"d:\\coding\\case\\conftest.py\", line 16, in with_login_cpc\n    cpclogin.close()\n  File \"d:\\coding\\base\\box.py\", line 923, in close\n    self.base_driver.close_browser()\n  File \"d:\\coding\\base\\box.py\", line 277, in close_browser\n    self._base_driver.close()\n  File \"c:\\users\\dajun\\documents\\caipiao\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 688, in close\n    self.execute(Command.CLOSE)\n  File \"c:\\users\\dajun\\documents\\caipiao\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"c:\\users\\dajun\\documents\\caipiao\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1568472767050, "stop": 1568472767067}], "start": 1568472753244, "stop": 1568472767067}